### ISSUE TITLE
Unable to downgrade iPad3,1 to iOS 8.4.1

### ISSUE THREAD

**Issue by jamesravi** [LukeZGD/iOS-OTA-Downgrader#189]
<details>
<summary>Terminal output</summary>

```
******* iOS-OTA-Downgrader ******* 
 - Downgrader script by LukeZGD -  

[Log] Running on platform: linux (Linux Mint 20.3) 
[Log] Checking Internet connection... 
[Log] Finding device in Normal mode... 
[Log] iPad3,1 (iOS 9.3.5) connected in Normal mode. 
[Log] ECID: 2002864151848 
*** Main Menu *** 
[Input] Select an option: 
1) Downgrade Device	      4) Put Device in kDFU Mode
2) Save OTA Blobs	      5) (Re-)Install Dependencies
3) Create Custom IPSW	      6) (Any other key to exit)
#? 1

[Input] Select iOS version: 
1) iOS 8.4.1		    3) (Any other key to exit)
2) Other (use SHSH blobs)
#? 1
[Log] Select your options when asked. If unsure, go for the defaults (press Enter/Return). 

[Input] Jailbreak Option 
* When this option is enabled, your device will be jailbroken on restore. 
* This option is enabled by default (Y). 
[Input] Enable this option? (Y/n): n
[Log] Jailbreak option disabled by user. 

[Log] Saving iOS 8.4.1 blobs with tsschecker... 
tsschecker version: 0.378-39daf4d4267ec867cccc4ccdcedaa0f9115011b8
libfragmentzip version: 0.Debug-Build: Feb 10 2022 01:30:53-RELEASE
[TSSC] manually specified generator "0x1111111111111111"
[TSSC] manually specified ECID to use, parsed "2002864151848" to dec:2002864151848 hex:1d254019d28
[TSSC] opening resources/manifests/BuildManifest_iPad3,1_8.4.1.plist
[TSSR] Checking BuildIdentity 0
[WARNING] [TSSC] could not get BuildIdentity for installType=Erase. Using fallback installType=Update since user did not specify installType manually
[TSSR] Checking BuildIdentity 0
[TSSR] Selected BuildIdentity for request
[TSSR] User specified to not request a baseband ticket.
[TSSR] Request URL set to https://gs.apple.com/TSS/controller?action=2
[TSSR] Sending TSS request attempt 1... success
[TSSR] Checking BuildIdentity 0
[Error] [TSSR] Error: could not get BuildIdentity for installType=Erase
Saved shsh blobs!

iOS 8.4.1 for device iPad3,1 IS being signed!
[Log] Successfully saved 8.4.1 blobs. 
[Log] Verifying iPad3,1_8.4.1_12H321_Restore.ipsw... 
[Log] Expected SHA1sum: c61165a638a16ddd9f607f589f1754613960c69a 
[Log] Actual SHA1sum:   c61165a638a16ddd9f607f589f1754613960c69a 
[Log] IPSW SHA1sum matches. 
[Log] Patching iBSS... 
[Log] Running iproxy for SSH... 
Creating listening port 2222 for device port 22
waiting for connection

[Log] Copying stuff to device via SSH... 
* Make sure OpenSSH/Dropbear is installed on the device and running! 
* Dropbear is only needed for devices on iOS 10 
* To make sure that SSH is successful, try these steps: 
* Reinstall OpenSSH/Dropbear, reboot and rejailbreak, then reinstall them again 

[Input] Enter the root password of your iOS device when prompted. 
* Note that you will be prompted twice. Do not worry that your input is not visible, it is still being entered. 
* The default password is "alpine" 
New connection for 2222->22, fd = 5
waiting for connection
Requesting connecion to USB device handle 1 (serial: e6432d8b30eef2e3dc15ad8121244472dd6736b8), port 22
Warning: Permanently added '[127.0.0.1]:2222' (RSA) to the list of known hosts.
root@127.0.0.1's password: 
kloader                                       100%   51KB   4.6MB/s   00:00    
pwnediBSS                                     100%   72KB   1.2MB/s   00:00    
[Log] Entering kDFU mode... 
* Press TOP or HOME button when the device disconnects and its screen goes black 
* You may also try to unplug and replug your device 
[Log] Finding device in DFU mode... 
New connection for 2222->22, fd = 5
waiting for connection
Requesting connecion to USB device handle 1 (serial: e6432d8b30eef2e3dc15ad8121244472dd6736b8), port 22
Warning: Permanently added '[127.0.0.1]:2222' (RSA) to the list of known hosts.
root@127.0.0.1's password: 
Connection to 127.0.0.1 closed by remote host.
[Log] Found device in DFU mode. 
[Log] Setting restore IPSW to: iPad3,1_8.4.1_12H321_Restore.ipsw 
[Log] Extracting IPSW: iPad3,1_8.4.1_12H321_Restore.ipsw 
[Log] Proceeding to futurerestore... 
[Log] Device iPad3,1 has no baseband 
[Log] Running futurerestore with command: ./resources/tools/futurerestore_linux -t "2002864151848_iPad3,1_j1ap_8.4.1-12H321_3a88b7c3802f2f0510abc432104a15ebd8bd7154.shsh2" --use-pwndfu --no-ibss --no-baseband "iPad3,1_8.4.1_12H321_Restore.ipsw" 
Version: v2.0.0-test(3bfba667cce95a85ec918210f8b05d2aed6c3d91-281)
img4tool version: 0.197-aca6cf005c94caf135023263cbb5c61a0081804f-RELEASE
libipatcher version: 0.88-1e855d70c84419014e363bdbcaead7b145fe3e1f-RELEASE
Odysseus for 32-bit support: yes
Odysseus for 64-bit support: yes
Serving HTTP on 0.0.0.0 port 8888 ...
[INFO] 32-bit device detected
futurerestore init done
reading signing ticket 2002864151848_iPad3,1_j1ap_8.4.1-12H321_3a88b7c3802f2f0510abc432104a15ebd8bd7154.shsh2 is done

WARNING: user specified is not to flash a baseband. This can make the restore fail if the device needs a baseband!
if you added this flag by mistake, you can press CTRL-C now to cancel
continuing restore in 10 9 8 7 6 5 4 3 2 1 
Found device in DFU mode
requesting to get into pwnRecovery later
Found device in DFU mode
Identified device as j1ap, iPad3,1
Extracting BuildManifest from iPSW
Product version: 8.4.1
Product build: 12H321 Major: 12
Device supports Image4: false
[TSSR] Checking BuildIdentity 0
[TSSR] Selected BuildIdentity for request
checking if the APTicket is valid for this restore...
Verified ECID in APTicket matches the device's ECID
[WARNING] skipping ramdisk hash check, since device is in pwnDFU according to user
Variant: Customer Erase Install (IPSW)
This restore will erase your device data.
Device found in DFU Mode.
Sending iBEC (295132 bytes)...
[==================================================] 100.0%
Booting iBEC, waiting for device to disconnect...
Booting iBEC, waiting for device to reconnect...
INFO: device serial number is DYVJ15QBDJ8T
Using cached filesystem from 'iPad3,1_8.4.1_12H321_Restore/058-24578-024.dmg'
Sending APTicket (2738 bytes)
Getting ApNonce in recovery mode... d0 81 91 31 6f f6 55 99 ea 55 75 ee 82 9f 81 5a 00 71 ae a9 
Sending APTicket (2738 bytes)
Recovery Mode Environment:
iBoot build-version=iBoot-2261.30.37
iBoot build-style=RELEASE
Sending RestoreLogo...
Extracting applelogo@2x~ipad.s5l8945x.img3 (Firmware/all_flash/all_flash.j1ap.production/applelogo@2x~ipad.s5l8945x.img3)...
Not personalizing component RestoreLogo...
Sending RestoreLogo (17868 bytes)...
ramdisk-size=0x4000000
Extracting 058-23954-023.dmg (058-23954-023.dmg)...
Not personalizing component RestoreRamDisk...
Sending RestoreRamDisk (16621836 bytes)...
Extracting DeviceTree.j1ap.img3 (Firmware/all_flash/all_flash.j1ap.production/DeviceTree.j1ap.img3)...
Not personalizing component RestoreDeviceTree...
Sending RestoreDeviceTree (83468 bytes)...
Extracting kernelcache.release.j1 (kernelcache.release.j1)...
Not personalizing component RestoreKernelCache...
Sending RestoreKernelCache (9257420 bytes)...
ERROR: Failed to place device in restore mode
Cleaning up...
[exception]:
what=ERROR: Unable to place device into restore mode

code=89325622
line=1363
file=/tmp/Builder/repos/futurerestore/src/futurerestore.cpp
commit count=281:
commit sha  =3bfba667cce95a85ec918210f8b05d2aed6c3d91:
Done: restoring failed!

[Log] Restoring done! 
[Log] Downgrade script done! 
```
</details>

**Comment by LukeZGD** [comment]
This might be caused by updating futurerestore, try this previous commit f0870cd: https://github.com/LukeZGD/iOS-OTA-Downgrader/archive/f0870cd9ce7a515fe443cc4430c5fd034b6d4c23.zip

**Comment by LukeZGD** [comment]
Fixed in 28467a1

**Comment by jamesravi** [comment]
Confirmed, it works now. Thanks a lot!
